qa/rpc-tests/test_framework/util.py

```diff
@@ -228,7 +260,7 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
     if binary is None:
         binary = os.getenv("BITCOIND", "bitcoind")
     # RPC tests still depend on free transactions
-    args = [ binary, "-datadir="+datadir, "-server", "-keypool=1", "-discover=0", "-rest", "-blockprioritysize=50000" ]
+    args = [ binary, "-datadir="+datadir, "-keypool=1", "-rest"] # , "-blockprioritysize=50000" ]
```

Comment:
Remove dead code. Maybe leave a comment if you think something about blockprioritysize can influence the test, or not.